### PR TITLE
90 speaking  using polly for text to speech

### PR DESCRIPTION
**Pull request by sahmedkh**
Implementing Polly for text to speech
Note: I am assuming that the key stored in DynamoDB for the Polly recording will be stored when the questions are generated. Furthermore, to test this you need to copy the objects in the `speaking-questions-polly` bucket to the `speakingpollybucket` in your stage.

### CODE REVIEW

**Comment by mzhubail** [comment]
@sahmedkh can you please elaborate on `speaking-questions-polly` and `speakingpollybucket`.

I see `speakingPollyBucket` in 6d59ef, but I couldn't find the other bucket.  What are the objects in each bucket and why is it necessary to copy them over by hand?

File: stacks/DBStack.ts [hunk 1]
```diff
@@ -18,11 +17,15 @@ export function DBStack(this: any, { stack }: StackContext) {
...
-    partitionKey: { name: 'MyPartitionKey', type: dynamodb.AttributeType.STRING },
+    partitionKey: {
+      name: 'MyPartitionKey',
+      type: dynamodb.AttributeType.STRING,
+    },
     sortKey: { name: 'MySortKey', type: dynamodb.AttributeType.STRING },
   });
 
   const uploads_bucket = new Bucket(stack, 'Uploads');
+  const speakingPollyBucket = new Bucket(stack, 'speakingPolly');
```
Comment (mzhubail) [inline stacks/DBStack.ts:28]: Why not use the existing bucket?

```suggestion

  /* For now we;re using a bucket that is already filled
   * TODO: change this when the feature of adding questions is implemented
   */
  const speakingPollyBucket = s3.Bucket.fromBucketAttributes(
    this,
    'speakingPolly',
    {
      bucketArn: 'arn:aws:s3:::speaking-questions-polly',
    },
  );
  // const speakingPollyBucket = new Bucket(stack, 'speakingPolly');
```

use this import

```ts
import * as s3 from 'aws-cdk-lib/aws-s3';
```

Comment (sahmedkh) [reply stacks/DBStack.ts:28]: Not a bad idea, I will try that now

**Review by mzhubail — APPROVED** [review]
Good job @sahmedkh!

**Pull request merged**